filebeat/prospector/redis/doc.go

```diff
@@ -0,0 +1,14 @@
+// Redis package contains prospector and harvester to read the redis slow log
+//
+// The redis slow log is stored in memory. The slow log can be activate on the redis command line as following:
+//
+// 	CONFIG SET slowlog-log-slower-than 100
+//
+// This sets the value of the slow log to 100 micro seconds. All queries taking longer will be reported.
```

Comment:
I'd use 2s or 5s as an example. 100 micros might affect the Redis performance, so we should be careful.